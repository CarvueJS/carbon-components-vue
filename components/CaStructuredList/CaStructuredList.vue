<template>
  <section class="bx--structured-list">
    <slot></slot>
  </section>
</template>

<script>
  export default {
    name: 'ca-structured-list'
  }
</script>

<style></style>
