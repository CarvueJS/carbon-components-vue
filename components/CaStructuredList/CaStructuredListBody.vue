<template>
  <div class="bx--structured-list-tbody">
    <slot></slot>
  </div>
</template>

<script>
  export default {
    name: 'ca-structured-list-body'
  }
</script>

<style></style>
