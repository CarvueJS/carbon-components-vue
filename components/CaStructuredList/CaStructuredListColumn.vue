<template>
  <div class="bx--structured-list-td">
    <slot></slot>
  </div>
</template>

<script>
  export default {
    name: 'ca-structured-list-col'
  }
</script>

<style></style>
