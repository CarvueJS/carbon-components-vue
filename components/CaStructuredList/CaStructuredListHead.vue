<template>
  <div class="bx--structured-list-thead">
    <slot></slot>
  </div>
</template>

<script>
  export default {
    name: 'ca-structured-list-head'
  }
</script>

<style></style>
