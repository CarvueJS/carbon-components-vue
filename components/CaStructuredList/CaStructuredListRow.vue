<template>
  <div class="bx--structured-list-row">
    <slot></slot>
  </div>
</template>

<script>
export default {
  name: 'ca-structured-list-row',
};
</script>

<style></style>
