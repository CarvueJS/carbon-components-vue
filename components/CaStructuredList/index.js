import CaStructuredList from './CaStructuredList.vue';
import CaStructuredListBody from './CaStructuredListBody.vue';
import CaStructuredListColumn from './CaStructuredListColumn.vue';
import CaStructuredListHead from './CaStructuredListHead.vue';
import CaStructuredListRow from './CaStructuredListRow.vue';

CaStructuredList.install = (Vue) => {
  Vue.component(CaStructuredList.name, CaStructuredList);
  Vue.component(CaStructuredListBody.name, CaStructuredListBody);
  Vue.component(CaStructuredListColumn.name, CaStructuredListColumn);
  Vue.component(CaStructuredListHead.name, CaStructuredListHead);
  Vue.component(CaStructuredListRow.name, CaStructuredListRow);
};

export default CaStructuredList;
