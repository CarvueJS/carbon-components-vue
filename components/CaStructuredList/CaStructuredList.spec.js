import { shallow } from 'vue-test-utils';
import CaStructuredList from './CaStructuredList.vue';

describe('CaStructuredList', () => {
  test('is a Vue instance', () => {
    const wrapper = shallow(CaStructuredList);
    expect(wrapper.isVueInstance()).toBeTruthy();
  });

  test('Mathes snapshot', () => {
    const wrapper = shallow(CaStructuredList);
    expect(wrapper.html()).toMatchSnapshot()
  });
})
