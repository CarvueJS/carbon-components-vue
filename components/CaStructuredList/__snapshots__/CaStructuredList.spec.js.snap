// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CaStructuredList Mathes snapshot 1`] = `<section class="bx--structured-list"></section>`;
